accepted/00075-live-patching-enhancements.md

```diff
@@ -0,0 +1,170 @@
+ - `zypper lifecycle` currently provides no machine-readable format
+ - Extra packages need to be installed on the client:
+   - `zypper-lifecycle-plugin`
+   - `lifecycle-data-sle-module-live-patching`
+
+#### Read the lifecycle data file on client
+
+Uyuni runs a Salt state to read the lifecycle data file provided with the package `lifecycle-data-sle-module-live-patching` on the client and collects the data during package profile update. The data is then stored in the database per client along with the kernel live version data.
+
+**Pros:**
+ - Easy to implement
+ - Can be attached to profile update data with an existing process
+
+**Cons:**
+ - File name or format can change in the future
+ - `lifecycle-data-sle-module-live-patching` package must be installed on the client
+
+#### Unpack and read the lifecycle data RPM on Uyuni server
+
+The lifecycle information is provided in the `lifecycle-data-sle-module-live-patching` package in the Live Patching product. Uyuni unpacks this RPM and reads the data from the extracted CSV file during reposync of this product. The data is stored in the database per live patch package.
+
+**Pros:**
+ - Independent, no impositions on clients
+ - Gathered data is always available and can be used for UI enhancements such as displaying EOL info for all live patch packages displayed in the UI
+
+**Cons:**
+ - RPM format can change in the future
+ - Uyuni doesn't have any existing functionality that can be utilized
+ - Unpacking must be done during reposync, adding complextion
+
+## 2. Live patching CLM workflow
+
+SLES channels can be prepared for live patching applications using the Content Lifecycle Management feature in Uyuni. This is done by adding a set of filters that exclude the regular kernel package updates and include live patch versions of those. However, the UI doesn't provide any special workflow for live patching and this process must be done manually by the user.
+
+Below are some examples of such sets of filters:
+
+Live patching project with a single filter to exclude regular kernel packages newer than a specific version:
+ - `DENY erratum contains package with epoch/version/release greater than kernel-default <EVR> (package_nevr)`
+
+Live patching project using `ALLOW` overrides:
+ - `DENY errata containing "update for the Linux Kernel" in synopsis`
+ - `ALLOW errata containing "Live Patch" in synopsis` *- an override to the first rule*
+ - `ALLOW a specific patch containing current kernel version (via advisory name)` *- an override to the first rule*
+
+### CLM project wizards
+
+The process of live patching can be made more user-friendly using "CLM project wizards". The wizards are alternatives to the regular filter creation dialogs that provide a tailored interaction to create multiple filters that serve a single purpose. These dialogs can be accessed via the "Use wizard" button in the filter creation dialog. This concept can be extended to other use cases and workflows in the future.
```

Comment:
This is a bit unclear. When we talk about a "Project Wizard" it might create the whole project incl. Channels, Environments, Filters, etc. . 
But when the wizard should only create filters, we should name it "Filter Wizard".

This become important below.

Reply:
I agree. "Filter wizard" sounds more suitable for what I had in mind. I'll change it accordingly.